### PR TITLE
Use Snake Case Naming Convention for Some Test Names

### PR DESCRIPTION
**Pull request by shiki**
Continuation of #2601. 

This updates all of StorageTests and some in NetworkingTests to use snake-case naming convention. 

## Testing

The build passing should be good enough. 🙂 

## Submitter Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Networking/NetworkingTests/Mapper/AccountMapperTests.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ class AccountMapperTests: XCTestCase {
 
     /// Verifies that all of the Account fields are properly parsed.
     ///
-    func testAccountFieldsArePropertlyParsed() {
+    func test_Account_fields_are_propertly_parsed() {
```
Comment (pmusolino) [inline Networking/NetworkingTests/Mapper/AccountMapperTests.swift:11]:
```suggestion
    func test_Account_fields_are_properly_parsed() {
```

File: Networking/NetworkingTests/Mapper/AccountSettingsMapperTests.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@ class AccountSettingsMapperTests: XCTestCase {
 
     /// Verifies that all of the AccountSettings fields are properly parsed.
     ///
-    func testAccountFieldsArePropertlyParsed() {
+    func test_Account_fields_are_propertly_parsed() {
```
Comment (pmusolino) [inline Networking/NetworkingTests/Mapper/AccountSettingsMapperTests.swift:11]:
```suggestion
    func test_Account_fields_are_properly_parsed() {
```

File: Networking/NetworkingTests/Mapper/ProductListMapperTests.swift [hunk 3]
```diff
@@ -92,7 +92,7 @@ class ProductListMapperTests: XCTestCase {
 
     /// Test that ProductTypeKey converts to
     /// a ProductType enum properly.
-    func testThatProductTypeKeyConvertsToEnumProperly() {
+    func test_that_product_type_key_converts_to_enum_properly() {
```
Comment (pmusolino) [inline Networking/NetworkingTests/Mapper/ProductListMapperTests.swift:95]: Maybe `test_that_productTypeKey_converts_to_enum_properly`?
```suggestion
    func test_that_productTypeKey_converts_to_enum_properly() {
```

**Review by pmusolino — APPROVED** [review]
Looks good to me! :shipit: 

Just two minor fix (typos) and one suggestion 👍

File: Networking/NetworkingTests/Mapper/ProductListMapperTests.swift (hunk 3 above)
Comment (shiki) [reply Networking/NetworkingTests/Mapper/ProductListMapperTests.swift]: Good catch! Fixed in 133487a00c64f3f3ece7240be31e1c0af5084dd0.

**Pull request merged**